static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -66,14 +66,22 @@ require.def(
                 SUBTITLE : 19,
                 CUSTOM : 20
             },
+            PlayerEmps: {
+                Player : 0,
+                StreamingPlayer : 1
+            },
 
             init: function() {
                 this._super();
                 this._state = MediaPlayer.STATE.EMPTY;
+                this._currentPlayer = undefined;
                 this._deferSeekingTo = null;
+                this._nextSeekingTo = null;
                 this._postBufferingState = null;
                 this._tryingToPause = false;
                 this._currentTimeKnown = false;
+                this._updatingTime = false;
+                this._lastWindowRanged = false;
 
                 try {
                     this._registerSamsungPlugins();
```

Comment:
If error is not ignored then it is necessary to add extra mock for tvmwplugin in unit tests.